app/update.go

```diff
@@ -545,45 +551,54 @@ func (m *Model) removeCachedItem(ts string) {
 	}
 }
 
+// enable/disable the main keys relevant to the preview view
 func (m *Model) setPreviewKeys(v bool) {
-	m.list.KeyMap.CursorUp.SetEnabled(v)
-	m.list.KeyMap.CursorDown.SetEnabled(v)
-	m.list.KeyMap.Filter.SetEnabled(v)
-	m.list.KeyMap.GoToEnd.SetEnabled(v)
-	m.list.KeyMap.GoToStart.SetEnabled(v)
-	m.list.KeyMap.Quit.SetEnabled(v)
-	m.list.KeyMap.NextPage.SetEnabled(v)
-	m.list.KeyMap.PrevPage.SetEnabled(v)
-	m.list.KeyMap.ShowFullHelp.SetEnabled(v)
-
-	m.keys.remove.SetEnabled(v)
-	m.keys.choose.SetEnabled(v)
-	m.keys.togglePin.SetEnabled(v)
-	m.keys.togglePinned.SetEnabled(v)
-	m.keys.selectDown.SetEnabled(v)
-	m.keys.selectUp.SetEnabled(v)
-	m.keys.selectSingle.SetEnabled(v)
-	m.keys.clearSelected.SetEnabled(v)
+	m.list.KeyMap.CursorUp.SetEnabled(!v)
+	m.list.KeyMap.CursorDown.SetEnabled(!v)
+	m.list.KeyMap.Filter.SetEnabled(!v)
+	m.list.KeyMap.GoToEnd.SetEnabled(!v)
+	m.list.KeyMap.GoToStart.SetEnabled(!v)
+	m.list.KeyMap.Quit.SetEnabled(!v)
+	m.list.KeyMap.NextPage.SetEnabled(!v)
+	m.list.KeyMap.PrevPage.SetEnabled(!v)
+	m.list.KeyMap.ShowFullHelp.SetEnabled(!v)
+
+	m.keys.remove.SetEnabled(!v)
+	m.keys.choose.SetEnabled(!v)
+	m.keys.togglePin.SetEnabled(!v)
+	m.keys.togglePinned.SetEnabled(!v)
+	m.keys.selectDown.SetEnabled(!v)
+	m.keys.selectUp.SetEnabled(!v)
+	m.keys.selectSingle.SetEnabled(!v)
+	m.keys.clearSelected.SetEnabled(!v)
```

Comment:
This is a source of bug I think

If one ! is forgotten on one line it won't be noticed.

I would define

```go
newValue := !v
```

Then use `newValue` It would be clearer and safer I think

Reply:
@ccoVeille Agreed! Will get that updated :blue_heart: